views/old_settings.html

```diff
@@ -114,12 +109,6 @@ <h2 class="nav-tab-wrapper" id="mailpoet_settings_tabs">
           if ($('.mailpoet_mta_setup_save').is(':visible')) {
             $('.mailpoet_mta_setup_save').trigger('click');
           }
-          var mailpoet_premium_key = $('#mailpoet_premium_key').val();
-          // sync mss key with premium key
-          $('#mailpoet_api_key').val(mailpoet_premium_key);
-          if (mailpoet_premium_key.length > 0) {
-            $('#mailpoet_premium_key_verify').trigger('click', false);
-          }
```

Comment:
Interesting. Will we use only a single key in the settings store? It actually seems like we used it like that already before... just in DB there seem to be 2 keys. So I guess this is OK since [we indeed save it for MSS as well](https://github.com/mailpoet/mailpoet/pull/2685/files#diff-2909d3626e5a2ce4276dfd5d29be6628R57).